### PR TITLE
work on both commands packages

### PR DESCRIPTION
**Pull request by Machine-Maker**

### CODE REVIEW

File: data/net/minecraft/commands/arguments/ArgumentSignatures.mapping [hunk 1]
```diff
@@ -3,6 +3,11 @@ CLASS net/minecraft/commands/arguments/ArgumentSignatures
 		ARG 1 buffer
 	METHOD get (Ljava/lang/String;)Lnet/minecraft/network/chat/MessageSignature;
 		ARG 1 key
+	METHOD lambda$signCommand$1 (Lnet/minecraft/commands/arguments/ArgumentSignatures$Signer;Lnet/minecraft/network/chat/SignableCommand$Argument;)Lnet/minecraft/commands/arguments/ArgumentSignatures$Entry;
+		ARG 1 signedArgument
+	METHOD lambda$write$0 (Lnet/minecraft/network/FriendlyByteBuf;Lnet/minecraft/commands/arguments/ArgumentSignatures$Entry;)V
+		ARG 0 byteBuf
```
Comment (alcatrazEscapee) [inline data/net/minecraft/commands/arguments/ArgumentSignatures.mapping]:
```suggestion
		ARG 0 buffer
```

As per previously established conventions, [here](https://github.com/ParchmentMC/Parchment/pull/207#discussion_r1248014538), and [here](https://github.com/ParchmentMC/Parchment/pull/24#pullrequestreview-695217286)

File: data/net/minecraft/server/commands/ResetChunksCommand.mapping [hunk 2]
```diff
@@ -1,4 +1,20 @@
 CLASS net/minecraft/server/commands/ResetChunksCommand
+	METHOD lambda$register$0 (Lnet/minecraft/commands/CommandSourceStack;)Z
+		ARG 0 soruce
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/ResetChunksCommand.mapping]:
```suggestion
		ARG 0 source
```

File: data/net/minecraft/commands/arguments/GameModeArgument.mapping [hunk 3]
```diff
@@ -2,6 +2,8 @@ CLASS net/minecraft/commands/arguments/GameModeArgument
 	METHOD getGameMode (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lnet/minecraft/world/level/GameType;
 		ARG 0 context
 		ARG 1 name
+	METHOD lambda$static$0 (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 unknownGameType
```
Comment (alcatrazEscapee) [inline data/net/minecraft/commands/arguments/GameModeArgument.mapping]:
```suggestion
		ARG 0 invalidGameType
```

Based on the field `ERROR_INVALID` and translation `"argument.gamemode.invalid"` using `invalid`, not `unknown`.

File: data/net/minecraft/server/commands/ScheduleCommand.mapping [hunk 4]
```diff
@@ -1,4 +1,23 @@
...
+	METHOD lambda$register$3 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD lambda$register$4 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD lambda$register$5 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD lambda$register$6 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD lambda$schedule$10 (Lnet/minecraft/resources/ResourceLocation;ZLnet/minecraft/world/level/timers/TimerQueue;JLnet/minecraft/commands/CommandSourceStack;ILjava/util/Collection;)V
+		ARG 7 functionCollection
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/ScheduleCommand.mapping]:
```suggestion
		ARG 7 functions
```

imo `functionCollection` is ungangly, this describes what it is better (either a single function, or functions plural). The type in question is the right of a `Either<CommandFunction, Collection<CommandFunction>>>`

**Review by alcatrazEscapee — CHANGES_REQUESTED** [review]
Overall looks good, a couple comments and one typo.

**Comment by Machine-Maker** [comment]
Ok, both commands packages should be completely done now except for methods that override things, like ArgumentType from brig.

File: data/net/minecraft/server/commands/EnchantCommand.mapping [hunk 5]
```diff
@@ -4,6 +4,21 @@ CLASS net/minecraft/server/commands/EnchantCommand
...
+		ARG 0 context
+	METHOD lambda$static$0 (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 entity
+	METHOD lambda$static$1 (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 entity
+	METHOD lambda$static$2 (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 item
+	METHOD lambda$static$3 (Ljava/lang/Object;Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 requestedLvl
+		ARG 1 maxLvl
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/EnchantCommand.mapping]:
```suggestion
		ARG 0 requestedLevel
		ARG 1 maxLevel
```
Don't worry we can afford to pay extra for the vowels here.

File: data/net/minecraft/server/commands/ExecuteCommand.mapping [hunk 6]
```diff
@@ -59,6 +59,163 @@ CLASS net/minecraft/server/commands/ExecuteCommand
...
+	METHOD lambda$addConditionals$54 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD lambda$addConditionals$55 (Lcom/mojang/brigadier/context/CommandContext;)Z
+		ARG 0 context
+	METHOD lambda$addConditionals$56 (ZLnet/minecraft/server/commands/data/DataCommands$DataProvider;Lcom/mojang/brigadier/context/CommandContext;)Ljava/util/Collection;
+		ARG 2 context
+	METHOD lambda$addConditionals$57 (Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 1 context
+	METHOD lambda$addConditionals$58 (Lcom/mojang/brigadier/tree/CommandNode;ZLnet/minecraft/server/commands/data/DataCommands$DataProvider;Lcom/mojang/brigadier/builder/ArgumentBuilder;)Lcom/mojang/brigadier/builder/ArgumentBuilder;
+		ARG 3 argBuilder
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/ExecuteCommand.mapping]:
```suggestion
		ARG 3 builder
```

Would prefer `argumentBuilder` or `builder` over the not-a-full-word

File: data/net/minecraft/server/commands/ExperienceCommand.mapping [hunk 7]
```diff
@@ -21,3 +41,13 @@ CLASS net/minecraft/server/commands/ExperienceCommand
 			ARG 4 add
 			ARG 5 set
 			ARG 6 query
+		METHOD lambda$static$0 (Lnet/minecraft/server/level/ServerPlayer;Ljava/lang/Integer;)Z
+			ARG 0 player
+			ARG 1 xp
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/ExperienceCommand.mapping]:
```suggestion
			ARG 1 experiencePoints
```

File: data/net/minecraft/server/commands/ExperienceCommand.mapping [hunk 8]
```diff
@@ -21,3 +41,13 @@ CLASS net/minecraft/server/commands/ExperienceCommand
...
+		METHOD lambda$static$1 (Lnet/minecraft/server/level/ServerPlayer;)I
+			ARG 0 player
+		METHOD lambda$static$2 (Lnet/minecraft/server/level/ServerPlayer;Ljava/lang/Integer;)Z
+			ARG 0 player
+			ARG 1 xp
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/ExperienceCommand.mapping]:
```suggestion
			ARG 1 experienceLevels
```

Unless I'm mistaken, these two lambdas are referring to different values (as per `Type`) and this is a clearer name for each.

File: data/net/minecraft/server/commands/JfrCommand.mapping [hunk 9]
```diff
@@ -1,4 +1,14 @@
 CLASS net/minecraft/server/commands/JfrCommand
+	METHOD lambda$register$1 (Lnet/minecraft/commands/CommandSourceStack;)Z
+		ARG 0 source
+	METHOD lambda$register$2 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD lambda$register$3 (Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 0 context
+	METHOD lambda$static$0 (Ljava/lang/Object;)Lcom/mojang/brigadier/Message;
+		ARG 0 msg
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/JfrCommand.mapping]:
```suggestion
		ARG 0 message
```

File: data/net/minecraft/server/commands/LootCommand.mapping [hunk 10]
```diff
@@ -68,8 +68,74 @@ CLASS net/minecraft/server/commands/LootCommand
...
+	METHOD lambda$register$10 (Lnet/minecraft/server/commands/LootCommand$DropConsumer;Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 1 context
+	METHOD lambda$register$11 (Lnet/minecraft/server/commands/LootCommand$DropConsumer;Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 1 context
+	METHOD lambda$register$12 (Lnet/minecraft/server/commands/LootCommand$DropConsumer;Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 1 context
+	METHOD lambda$register$13 (Lnet/minecraft/server/commands/LootCommand$DropConsumer;Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 1 context
+	METHOD lambda$register$14 (Lnet/minecraft/commands/CommandBuildContext;Lcom/mojang/brigadier/builder/ArgumentBuilder;Lnet/minecraft/server/commands/LootCommand$DropConsumer;)Lcom/mojang/brigadier/builder/ArgumentBuilder;
+		ARG 1 argBuilder
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/LootCommand.mapping]:
```suggestion
		ARG 1 builder
```

File: data/net/minecraft/server/commands/LootCommand.mapping [hunk 11]
```diff
@@ -93,4 +159,5 @@ CLASS net/minecraft/server/commands/LootCommand
 			ARG 3 callback
 	CLASS TailProvider
 		METHOD construct (Lcom/mojang/brigadier/builder/ArgumentBuilder;Lnet/minecraft/server/commands/LootCommand$DropConsumer;)Lcom/mojang/brigadier/builder/ArgumentBuilder;
+			ARG 1 argBuilder
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/LootCommand.mapping]:
```suggestion
			ARG 1 builder
```

File: data/net/minecraft/server/commands/TeleportCommand.mapping [hunk 12]
```diff
@@ -1,6 +1,26 @@
 CLASS net/minecraft/server/commands/TeleportCommand
 	METHOD formatDouble (D)Ljava/lang/String;
 		ARG 0 value
+	METHOD lambda$register$0 (Lnet/minecraft/commands/CommandSourceStack;)Z
+		ARG 0 soruce
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/TeleportCommand.mapping]:
```suggestion
		ARG 0 source
```

File: data/net/minecraft/server/commands/data/BlockDataAccessor.mapping [hunk 13]
```diff
@@ -10,5 +10,7 @@ CLASS net/minecraft/server/commands/data/BlockDataAccessor
 	METHOD getPrintSuccess (Lnet/minecraft/nbt/Tag;)Lnet/minecraft/network/chat/Component;
 		COMMENT Gets the message used as a result of querying the given NBT (both for /data get and /data get path)
 		ARG 1 nbt
+	METHOD lambda$static$0 (Ljava/lang/String;)Lnet/minecraft/server/commands/data/DataCommands$DataProvider;
+		ARG 0 argName
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/data/BlockDataAccessor.mapping]:
```suggestion
		ARG 0 name
```

File: data/net/minecraft/server/commands/data/DataCommands.mapping [hunk 14]
```diff
@@ -33,6 +33,101 @@ CLASS net/minecraft/server/commands/data/DataCommands
...
+		ARG 3 context
+	METHOD lambda$decorateModification$23 (Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lnet/minecraft/server/commands/data/DataCommands$DataManipulator;Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lcom/mojang/brigadier/context/CommandContext;)I
+		ARG 3 context
+	METHOD lambda$decorateModification$24 (Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lnet/minecraft/server/commands/data/DataCommands$DataManipulator;Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lcom/mojang/brigadier/builder/ArgumentBuilder;)Lcom/mojang/brigadier/builder/ArgumentBuilder;
+		ARG 3 subArgBuilder
+	METHOD lambda$decorateModification$25 (Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lnet/minecraft/server/commands/data/DataCommands$DataProvider;Lnet/minecraft/server/commands/data/DataCommands$DataManipulator;)Lcom/mojang/brigadier/builder/ArgumentBuilder;
+		ARG 2 dataManipulator
+	METHOD lambda$decorateModification$26 (Ljava/lang/String;)Ljava/lang/String;
+		ARG 0 str
```
Comment (alcatrazEscapee) [inline data/net/minecraft/server/commands/data/DataCommands.mapping]: I don't like `str` as a arbitrary parameter name, it tells very little (other than the type, badly), and is a abbreviation. For all the similarly named parameters in this class.

**Review by alcatrazEscapee — CHANGES_REQUESTED** [review]

**Review by alcatrazEscapee — APPROVED** [review]

**Pull request merged**